project/plugins.sbt

```diff
@@ -6,7 +8,7 @@ addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.0.0")
 
```

Comment:
Did you try "3.4.0" or "3.5.0" versions for sbt auto build?

Reply:
thats creating issues in this service with some other plugin , so not used the latest version of sbt-auto-build